.gitlab/llnl-ci.yml

```diff
@@ -0,0 +1,50 @@
+stages:
+  - build
+  - test
+
+Spack Build:
+  stage: build
+  tags:
+    - quartz
+    - shell
+  needs: []
+  script:
+    - cd $SPACK_WORK_DIR/spack/
+    - git pull
+    - . $SPACK_WORK_DIR/spack/share/spack/setup-env.sh
+    - spack uninstall -y exaworks py-pytest || true
+    - spack install --fresh exaworks py-pytest
+
+Spack Test:
+  variables:
+    LLNL_SLURM_SCHEDULER_PARAMETERS: "--nodes=1 -t 15"
+  stage: test
+  before_script:
+    - whoami
+  parallel:
+    matrix:
+      - HOST : ["quartz", "oslic", "ruby"]
```

Comment:
How come not Lassen or one of the other BlueOS machines?

Reply:
That's a good point, I tried running on Lassen but I think I need a different Spack Build. I figured I could look into it a bit more and add to it with a separate PR, but I am also open to adding it to this one

Reply:
I think it would be good to add at some point. I would probably do it as a separate PR though.